### DISCUSSION TITLE
syncing highlights

### DISCUSSION THREAD

**Discussion by tokisuno** [ahrm/sioyek#719] [Q&A] [answered]
yo yo

is there any way to sync highlights from computer to computer? or rather is there any way to save highlights for a specific file and have them sync over when you access it from another computer?

i am doing work with vim and latex, and i want to highlight portions of an academic paper i am translating for myself. but i want those highlights to be synced between my x220 and my desktop. i sync between devices in syncthing.

**Comment by ahrm** [comment, marked as answer]
https://sioyek-documentation.readthedocs.io/en/latest/usage.html#data-and-synchronization

There is also an `export` and `import` command to save/load data to/from a json file.

**Reply by tokisuno** [reply]
oh that's sick thanks ahrm
